src/EditorFeatures/Core/FindUsages/IDefinitionsAndReferencesFactory.cs

```diff
@@ -69,6 +69,17 @@ internal static class DefinitionItemExtensions
                 options: FindReferencesSearchOptions.Default, cancellationToken: CancellationToken.None).WaitAndGetResult_CanCallOnBackground(CancellationToken.None);
         }
 
+        public static Task<DefinitionItem> ToNonClassifiedDefinitionItemAsync(
+            this ISymbol definition,
+            Solution solution,
+            bool includeHiddenLocations,
+            CancellationToken cancellationToken)
+        {
+            return ToDefinitionItemAsync(
+                definition, solution, isPrimary: false, includeHiddenLocations, includeClassifiedSpans: false,
+                options: FindReferencesSearchOptions.Default, cancellationToken: cancellationToken);
```

Comment:
❔ What does `FindReferencesSearchOptions` get used for here? Is there any reason why we use `FindReferencesSearchOptions.Default` without setting `UnidirectionalHierarchyCascade` (the latter performs better)